securing_apps/topics/docker/quickstart-zip.adoc

```diff
@@ -0,0 +1,22 @@
+[[_docker_quickstart_zip]]
+
+=== Docker Registry Quickstart Zip
+
+WARNING: This installation method is meant to be an easy way to get a docker registry authenticating against a keycloak server.  It is intended for development purposes only and should never be used in a production or production-like environment.
+
+The zip file installation mechanism provides a quickstart for developers who want to understand how the keycloak server can interact with the docker registry.  In order to configure the quickstart:
+
+ 1. Begin by creating a client configuration.  At this point you won't have a docker registry - the quickstart will take care of that part.
+ 1. Choose the "Quickstart" option from the installation tab and download the .zip file
+ 1. Unzip the archive to the desired location, and open the directory.
+ 1. Start the docker registry with `docker-compose up`
+ 1. Configure the realm in which the docker client is configured to use HTTP/Basic authentication as the browser auth flow
```

Comment:
Note:  Authenticator to use is now 'Docker Authenticator', and not 'HTTP Basic Authentication'.